test/example_bot.gleam

```diff
@@ -177,7 +179,8 @@ fn event_handler(bot, packet: event_handler.Packet) {
 
       case interaction.d.data.name {
         "test" -> {
-          discord_gleam.interaction_reply_message(interaction, "test")
+          discord_gleam.interaction_reply_message(interaction, "test", True)
+          discord_gleam.interaction_reply_message(interaction, "test", False)
```

Comment:
You can only reply to a interaction once, maybe make two commands test and test2? where one is global and one is guild?

Reply:
doesnt matter :sippycup: